app/javascript/react/src/redux/welcome/greetingsSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+import fetchData from "./fetchAPi";
+
+const initialState = {
+  data: '',
+  isSuccess: false,
+  message: '',
+  loading: false
+};
+
+const randomMessage = createSlice({
+  name: 'RandomMessage',
+  initialState,
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchData.pending, (state) => {
+        state.loading = true;
+      })
+      .addCase(fetchData.fulfilled, (state, action) => {
+        state.loading = false;
+        // state.isSuccess = ture;
+        state.data = action.payload
+      })
+      .addCase(fetchData.rejected, (state, action) => {
+        state.message = action.payload;
+        state.loading = false;
+        // state.isSuccess = false;
```

Comment:
- You have done well so far, but please remove all commented code from your application, this would help make your code cleaner and easier to maintain. Please fix this 👍 👍